### DISCUSSION TITLE
Android 12 beta 4 is not supported

### DISCUSSION THREAD

**Discussion by Mygod** [shadowsocks/shadowsocks-android#2782] [General]
Do not upgrade to Android 12 beta 4 until Google releases a fix.

See: https://issuetracker.google.com/issues/196271165

**Comment by Mygod** [comment]
Although it seems [VPN Hotspot](https://github.com/Mygod/VPNHotspot) still works. 😋

**Comment by dev4u** [comment]
好像已经修复了，但是没对应版本的机型没验证过。

**Comment by iKirby** [comment]
可以确认 Android 12 Beta 4.1 修复了这个问题
